### PR TITLE
Add clothing icons

### PR DESCRIPTION
**Pull request by leeanndrees**
I've added clothing items and done a bunch of styling and other stuff:

- Added icons for clothing items (including fallback icons for items that don't have specific icons)
- Displayed clothing items in a collection view
- Added a background gradient (static for now, eventually will change w/ temperature)
- Used custom fonts

### CODE REVIEW

File: dressForTheWeather/Models/ClothingItem.swift [hunk 1]
```diff
@@ -7,11 +7,22 @@
 //
 
 import Foundation
+import UIKit
 
 class ClothingItem {
     var name: String
     var placement: [BodyPlacement]
     var tempRange: ClosedRange<Double>
+    private var fallbackImages: [UIImage?] = [UIImage(named: "sparkle1") ?? nil,
```
Comment (leeanndrees) [inline dressForTheWeather/Models/ClothingItem.swift:16]: I'm not sure this is the best way to deal with these optionals, but I did find some good info on why they _should_ be optionals, at least: https://github.com/mac-cain13/R.swift/issues/61

File: dressForTheWeather/View Controllers/RecommendationViewController.swift [hunk 2]
```diff
@@ -46,13 +56,14 @@ extension RecommendationViewController: RecommendationViewDelegate {
         let alertController = UIAlertController(title: "Error", message: description, preferredStyle: .alert)
         let cancelAction = UIAlertAction(title: "OK", style: .default, handler: nil)
         alertController.addAction(cancelAction)
-        present(alertController, animated: true)
+//        present(alertController, animated: true)
```
Comment (leeanndrees) [inline dressForTheWeather/View Controllers/RecommendationViewController.swift:59]: This is that alert that shows all the time, when we really don't need it to. I'll figure out why it's doing that.... later. :)

File: dressForTheWeather/Assets.xcassets/Clothing Icons/hat.imageset/Contents.json [hunk 3]
```diff
@@ -0,0 +1,21 @@
+{
```
Comment (levosins) [inline dressForTheWeather/Assets.xcassets/Clothing Icons/hat.imageset/Contents.json:1]: Just a reminder to change the images to PDFs (for single scale)  and resize them to be the largest possible size (example maybe for these icons the max size is 100 X 100)

File: dressForTheWeather/Models/ClothingItem.swift (hunk 1 above)
Comment (levosins) [reply dressForTheWeather/Models/ClothingItem.swift:16]: Maybe try something like this instead?

```
var image: UIImage? {
        guard let theImage = UIImage(named: name) else { return fallbackImage }
        return theImage
    }
    
    private let imageNames = ["sparkle1", "sparkle2", "sparkle3"]
    
    private var fallbackImage: UIImage? {
        let imageName = imageNames.randomElement() ?? ""
        return UIImage(named: imageName)
    }
```

**Review by levosins — CHANGES_REQUESTED** [review]

File: dressForTheWeather/Models/ClothingItem.swift (hunk 1 above)
Comment (leeanndrees) [reply dressForTheWeather/Models/ClothingItem.swift:16]: yep, I like this a lot better. Applied in c4a2f8619cdbae70fe0ac925df7ef9ad0a78de70

File: dressForTheWeather/Assets.xcassets/Clothing Icons/hat.imageset/Contents.json (hunk 3 above)
Comment (leeanndrees) [reply dressForTheWeather/Assets.xcassets/Clothing Icons/hat.imageset/Contents.json:1]: I'll do this as a separate task in a future PR

**Review by levosins — APPROVED** [review]

**Pull request merged**